float64 cruiseSpeed
int32 firmwareType
float64 hoverSpeed
Mission_item_struct[] items 
point_LatLonAltVel plannedHomePosition
int32 vehicleType
int32 version
